### PR TITLE
Remove hard-coded references to seattle.gov domains

### PR DESCRIPTION
**Pull request by bion**
Removes hard-coded references to seattle-specific domains in application code and moves them into the application.conf file

### CODE REVIEW

File: universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java [hunk 1]
```diff
@@ -115,10 +119,13 @@ public Result addApplicant(Long id, Http.Request request) {
       return redirect(
           routes.TrustedIntermediaryController.dashboard(Optional.empty(), Optional.empty()));
     } catch (EmailAddressExistsException e) {
+      String tiUrl = baseUrl + "/trustedIntermediaries";
```
Comment (ettengererin) [inline universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java:122]: readability nit: Use descriptive names to be clear to a new reader. I would change "tiURL" to "trustedIntermediaryUrl"

File: universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java [hunk 2]
```diff
@@ -66,7 +68,7 @@ public HtmlBundle getBundle(HtmlBundle bundle) {
         meta().attr("name", "viewport").attr("content", "width=device-width, initial-scale=1"));
 
     // Add the warning toast, only for staging
-    if (STAGING_HOST_NAME.equals(hostName)) {
+    if (isStaging) {
```
Comment (ettengererin) [inline universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java:71]: Optional: Why add this isStaging field if it is only used once? 

You could leave it as if(hostName.equals(stagingHostname)). I understand you might be going for clarity with isStaging field.

File: universal-application-tool-0.0.1/app/views/LoginForm.java [hunk 3]
```diff
@@ -28,11 +28,13 @@
 
   private final BaseHtmlLayout layout;
   private final Config config;
+  private final FakeAdminClient fakeAdminClient;
 
   @Inject
-  public LoginForm(BaseHtmlLayout layout, Config config) {
+  public LoginForm(BaseHtmlLayout layout, Config config, FakeAdminClient fakeAdminClient) {
```
Comment (ettengererin) [inline universal-application-tool-0.0.1/app/views/LoginForm.java:34]: Out of curiosity, can you explain why this change is necessary?

**Review by ettengererin — APPROVED** [review]

File: universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java (hunk 1 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/controllers/ti/TrustedIntermediaryController.java]: Done

File: universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java (hunk 2 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/BaseHtmlLayout.java:71]: `isStaging` seemed like it takes slightly less effort for the reader's eyes to parse when they read the method

File: universal-application-tool-0.0.1/app/views/LoginForm.java (hunk 3 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:34]: Prior to this change, `canEnable` was a static method that relied on a hard-coded value that is now provided via start-time configuration. Static methods can't access start-time configuration, so `canEnable` needed to become an instance method, requiring an instance of `fakeAdminClient` be injected here.

As an aside, I generally try to encourage devs working in OO languages to prefer instance methods for public interfaces for this exact reason.

Comment (ettengererin) [reply universal-application-tool-0.0.1/app/views/LoginForm.java:34]: Great, thanks so much for this explanation.
